### DISCUSSION TITLE
Disable entry splash animation on dynamic website

### DISCUSSION THREAD

**Discussion by NSOPORTEDESARROLLO** [flet-dev/flet#3634] [General]
Hi, I have a survey website using flet, it runs as dynamic site ( all on my server ), I run "python3 survey.py" and it  works fine but I want to delete the splash animation, I tried build command and publish but it create a static site only, it is too slow for mobile. Does anybody know how disable it on dynamic ?

**Comment by Muddassir-Farooq-official** [comment]
Same question
